admin/docker/postgis.sql

```diff
@@ -0,0 +1 @@
+CREATE EXTENSION IF NOT EXISTS postgis;
```

Comment:
I don't understand where is this file used ...

Reply:
Here: https://github.com/camptocamp/c2cgeoportal/pull/3170/commits/1413713e9ceb369f21901614ca4ca05d8fa2d528#diff-e57874b7bc46f16fae1fda4d23d18c67R33